bitrise.yml

```diff
@@ -167,22 +167,52 @@ workflows:
         - xcodebuild_options: '"COMPILER_INDEX_STORE_ENABLE=NO" "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "CODE_SIGNING_ALLOWED=NO"'
         - xctestrun: $BITRISE_TEST_BUNDLE_PATH/Fennec_UnitTest_iphonesimulator18.2-arm64.xctestrun
         - maximum_test_repetitions: 2
+    - save-spm-cache@1.3.1:
+        is_always_run: true
     - xcode-test-shard-calculation@0:
         inputs:
         - product_path: $BITRISE_XCTESTRUN_SMOKE_TEST_FILE_PATH
         - shard_count: 5
         - destination: $BITRISE_DESTINATION
         outputs:
-        - BITRISE_TEST_SHARDS_PATH: BITRISE_TEST_SHARDS_PATH_FIREFOX 
-        is_always_run: true
-    - save-spm-cache@1.3.1:
+        - BITRISE_TEST_SHARDS_PATH: BITRISE_TEST_SHARDS_PATH_FIREFOX
+    - script@1.2.1:
+        title: Conditionally prepare deploy artifacts
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            set -euo pipefail
+            # List all Bitrise related variables
+            env | grep 'BITRISE_' | sort
+            # Create a temporary list of deployable files
+            DEPLOY_LIST=""
+            if [ -n "${BITRISE_TEST_SHARDS_PATH_FIREFOX:-}" ]; then
+                echo "Content of bitrise test shards path"
+                ls -la $BITRISE_TEST_SHARDS_PATH_FIREFOX
+                DEPLOY_LIST+="$BITRISE_TEST_SHARDS_PATH_FIREFOX"$':BITRISE_TEST_SHARDS_PATH\n'
```

Comment:
I have tried both `BITRISE_TEST_SHARDS_PATH` and `BITRISE_TEST_SHARD_PATH`, but `deploy-to-bitrise` step does not upload the content of the directory.